src/components/__tests__/IssueSearchAdvanced.spec.ts

```diff
@@ -57,7 +49,8 @@ describe('IssueSearchAdvanced', () => {
     expect(searchButton.exists()).toBeTruthy();
     await searchButton.trigger('submit');
     await flushPromises();
-    const setFilterEvent = wrapper.emitted('set:filters');
-    expect(setFilterEvent[0][0]).toEqual({ acknowledgments__name: 'test' });
+    expect(useRouter().replace).toHaveBeenCalled();
+    expect(useRouter().replace.mock.calls[0][0])
+      .toStrictEqual({ query: { query: 'search', acknowledgments__name: 'test' } });
```

Comment:
I think the router is not being tested here, but maybe I am missing something?

Reply:
We don't need to test router. I think we just make sure route.replace is called with correct params.